openvm/tests/apc_builder_outputs/memcpy_block.txt

```diff
@@ -71,14 +71,15 @@ mult=is_valid * 1, args=[writes_aux__base__timestamp_lt_aux__lower_decomp__0_0,
 mult=is_valid * 1, args=[15360 * writes_aux__base__prev_timestamp_0 + 15360 * writes_aux__base__timestamp_lt_aux__lower_decomp__0_0 - (15360 * from_state__timestamp_0 + 15360), 12]
 mult=is_valid * 1, args=[reads_aux__0__base__timestamp_lt_aux__lower_decomp__0_2, 17]
 mult=is_valid * 1, args=[15360 * reads_aux__0__base__prev_timestamp_2 + 15360 * reads_aux__0__base__timestamp_lt_aux__lower_decomp__0_2 - (15360 * from_state__timestamp_0 + 76800), 12]
+mult=is_valid * 1, args=[cmp_result_1 + cmp_result_2 - 2 * cmp_result_1 * cmp_result_2, 1]
```

Comment:
Degree 4 is too high?

Reply:
Yes, we started using degree bound 3 instead of 5, because it's usually faster.